whois-update/src/main/java/net/ripe/db/whois/update/handler/validator/organisation/CountryRipeMaintainedValidator.java

```diff
@@ -0,0 +1,59 @@
+package net.ripe.db.whois.update.handler.validator.organisation;
+
+import com.google.common.collect.ImmutableList;
+import net.ripe.db.whois.common.rpsl.ObjectType;
+import net.ripe.db.whois.common.rpsl.RpslObject;
+import net.ripe.db.whois.update.authentication.Principal;
+import net.ripe.db.whois.update.authentication.Subject;
+import net.ripe.db.whois.update.domain.Action;
+import net.ripe.db.whois.update.domain.PreparedUpdate;
+import net.ripe.db.whois.update.domain.UpdateContext;
+import net.ripe.db.whois.update.domain.UpdateMessages;
+import net.ripe.db.whois.update.handler.validator.BusinessRuleValidator;
+import org.springframework.stereotype.Component;
+
+import static net.ripe.db.whois.common.rpsl.AttributeType.COUNTRY;
+import static net.ripe.db.whois.common.rpsl.ObjectType.ORGANISATION;
+import static net.ripe.db.whois.update.domain.Action.CREATE;
+import static net.ripe.db.whois.update.domain.Action.MODIFY;
+
+@Component
+public class CountryRipeMaintainedValidator implements BusinessRuleValidator {
+
+    private static final ImmutableList<Action> ACTIONS = ImmutableList.of(CREATE, MODIFY);
+    private static final ImmutableList<ObjectType> TYPES = ImmutableList.of(ORGANISATION);
+
+    @Override
+    public void validate(final PreparedUpdate update, final UpdateContext updateContext) {
+        final Subject subject = updateContext.getSubject(update);
+        if (subject.hasPrincipal(Principal.OVERRIDE_MAINTAINER) || subject.hasPrincipal(Principal.ALLOC_MAINTAINER)) {
```

Comment:
Use RS_MAINTAINER instead of ALLOC_MAINTAINER ?

Reply:
Done.